lib/openhab/log/logger.rb

```diff
@@ -22,9 +22,21 @@ class Logger
       #
       # Regex for matching internal calls in a stack trace
       #
-      INTERNAL_CALL_REGEX = %r{(openhab-scripting-.*/lib)|(org/jruby/)}.freeze
+      INTERNAL_CALL_REGEX = %r{(openhab-scripting-.*/lib)|org.jruby}.freeze
       private_constant :INTERNAL_CALL_REGEX
 
+      #
+      # Regex for matching internal calls in a java stack trace
+      #
+      EXCLUDED_JAVA_PACKAGES = /jdk.internal.reflect|java.lang.reflect|org.openhab|java.lang.Thread.run/.freeze
```

Comment:
Should the dots be escaped?
```suggestion
      EXCLUDED_JAVA_PACKAGES = /jdk\.internal\.reflect|java\.lang\.reflect|org\.openhab|java\.lang\.Thread\.run/.freeze
```